### PR TITLE
Update electronic-design.md

### PR DESCRIPTION
**Pull request by Gogopex**
- fixed a typo l.7
- rephrased the `Compatible MCUs`section; the grammar is now a bit smoother.

### CODE REVIEW

**Review by Simonbdy — APPROVED** [review]
Ok for me for the line 7.
For the line 22, I'll let James review as he rewrote this part earlier I reckon.

**Review by jlangbridge — CHANGES_REQUESTED** [review]
Suggest "yet supported" on line 22; -> "is not yet supported"

File: src/pages/low/hardware_topics/electronic-design.md [hunk 1]
```diff
@@ -19,6 +19,6 @@ Luos' One-wire official connector is: <a href="https://octopart.com/df11-4dp-2ds
 Luos' RS485 official connector is: <a href="https://octopart.com/df11-8dp-2ds%2824%29-hirose-39521447" target="_blank">*DF11-8DP-2DS*</a>.
 
 ## Compatible MCUs
-Luos manages any type of microcontrollers, but they need to be added manually to the library. If your microcontroller is not supported yet, please contact us:
+Luos can manage any type of microcontrollers as long as they are first added to the library manually. If your microcontroller is not supported yet, please contact us:
```
Comment (Gogopex) [inline src/pages/low/hardware_topics/electronic-design.md:22]:
```suggestion
Luos can manage any type of microcontrollers as long as they are first added to the library manually. If your microcontroller is not yet supported, please contact us:
```

Comment (Gogopex) [reply src/pages/low/hardware_topics/electronic-design.md:22]: @jlangbridge I do agree this flows better 👍  Feel free to commit the suggestion if this is what you meant 👍

**Pull request merged**